daemon/src/peer/gossip.rs

```diff
@@ -41,11 +41,7 @@ where
         rev: None,
         origin,
     }) {
-        Ok(()) => log::trace!(
-            "successfully queried for urn=`{}`, origin=`{:?}`",
-            urn,
-            origin
-        ),
-        Err(_payload) => log::warn!("failed to query for urn=`{}`, origin=`{:?}`", urn, origin),
+        Ok(()) => tracing::trace!(?urn, "successfully queried URN"),
+        Err(_payload) => tracing::warn!(?urn, "failed to query URN"),
```

Comment:
Any reason we removed the `origin` from the message?

Reply:
Fixed in e0f8ba665088506a52784ac5335cbb639d40818b